alf/environments/suite_carla.py

```diff
@@ -334,11 +335,15 @@ def _IMU_callback(weak_self, sensor_data):
         self = weak_self()
         if not self:
             return
+        if not math.isnan(sensor_data.compass):
+            self._compass = sensor_data.compass
```

Comment:
maybe initialize self._compass to some default value in __init__()?

Reply:
"self._compass  = 0" is there already

Reply:
Ah, missed that..  LG then.